handler/handler.go

```diff
@@ -130,19 +150,54 @@ func (h *Handler) login(w http.ResponseWriter, r *http.Request) {
 		resp.Error(w, err)
 		return
 	}
-	// generate tokens
+	// reset token sessions
+	player.SessionTs = time.Now().Unix()
+	if err = h.srv.UpdatePlayerSession(r.Context(), payload.Username, player.SessionTs); err != nil {
+		resp.Error(w, err)
+		return
+	}
+	// generate tokens based on new sessions
 	var (
-		accessToken auth.Token
-		// refreshToken auth.Token
+		accessToken  auth.Token
+		refreshToken auth.Token
 	)
-	if accessToken, err = h.token.Generate(r.Context(), *player); err != nil {
+	if accessToken, err = h.token.Generate(r.Context(), *player, accessTokenTTL); err != nil {
+		resp.Error(w, err)
+		return
+	}
+	if refreshToken, err = h.token.Generate(r.Context(), *player, refreshTokenTTL); err != nil {
 		resp.Error(w, err)
 		return
 	}
-	result := loginResponse{AccessToken: string(accessToken) /* RefreshToken: string(refreshToken) */}
+	ck := newAccessCookie(accessToken)
+	http.SetCookie(w, &ck)
+	ck = newRefreshCookie(refreshToken)
+	http.SetCookie(w, &ck)
+	result := messageResponse{Message: "Login successful"}
 	resp.JSON(w, result)
+}
+
+func (h *Handler) logout(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	player := Player(ctx)
+
+	if err := h.srv.UpdatePlayerSession(ctx, player.Username, player.SessionTs-1); err != nil {
```

Comment:
does this really logout the user?

from the PR's code I believe point 6,7 from the [proposal](https://github.com/kodekulture/wordle-server/issues/30#issuecomment-2571426368) aren't implemented

i.e we can't logout other sessions from the current session since 

1) `xts` isn't checked if the access token or refresh_token is valid

2) For logout looks like we need to set `xts` to `player.SessionTs-1`, therefore current token will have `ts` lesss than the actual one in DB

Reply:
1. xts is not checked and that can be confusing yes. But xts is exactly player.SessionTs and we check this value. So point 6 is implemented.
2. can you draft a test case. I think my tests covered all of these cases

Reply:
I think I understand your point, but ignore xts and focus on player.SessionTs which is actually in the token payload as well.

Reply:
https://github.com/kodekulture/wordle-server/blob/798f527fcf3f0a40ccfd392a1dfde381ca77cd29/handler/middleware_test.go#L64